src/main.cpp

```diff
@@ -618,9 +618,13 @@ int main(int argc, char**argv)
                                                                 load_monitor,
                                                                 hs_comm_monitor);
 
+  CassandraResolver* cass_resolver = new CassandraResolver(dns_resolver,
+                                                           af,
+                                                           options.http_blacklist_duration,
```

Comment:
http_blacklist_duration? Should we have a cassandra_blacklist_duration, or should we rename it to blacklist_duration?